metron-interface/metron-alerts/package.json

```diff
@@ -46,9 +46,7 @@
     "@types/ace": "0.0.32",
     "@types/es6-promise": "0.0.33",
     "@types/jasmine": "2.5.38",
-    "@types/moment": "^2.13.0",
     "@types/node": "~6.0.60",
-    "@types/pikaday-time": "^1.4.2",
```

Comment:
The reason why I removed this is because moment.js is the dependency of this to make it able to add the type `moment` to the return value of the `getMoment` method. Therefore moment.js has to be inside the node_modules folder but we wan't to avoid it to be there. 
I don't see any benefits of having the type definitions of pikaday-time in the code base.